src/assets.ts

```diff
@@ -730,17 +764,16 @@ export async function generateAssets(server: OmniSharpServer, selectedIndex?: nu
         let workspaceInformation = await serverUtils.requestWorkspaceInformation(server);
         if (workspaceInformation.MsBuild && workspaceInformation.MsBuild.Projects.length > 0) {
             const generator = new AssetGenerator(workspaceInformation);
-            let doOverwriteAssets = await shouldOverwriteAssets(generator);
-            if (doOverwriteAssets === 'Cancel') {
+            let doGenerateAssets = await shouldGenerateAssets(generator);
+            if (!doGenerateAssets) {
                 return; // user cancelled
             }
 
-            let overwrite = doOverwriteAssets === 'Yes';
-
             const operations: AssetOperations = {
                 addLaunchJson: generator.hasExecutableProjects(),
-                addTasksJson: overwrite,
-                updateTasksJson: !overwrite,
+                // TODO: this distinction does make sense anymore, we either Update, or do nothing.
+                addTasksJson: true,
+                updateTasksJson: false,
```

Comment:
Since these are constant now, lets go ahead and remove them.

Reply:
I have removed the comment, but kept the values as they were before. There is only addTasksJson as before.